spec/npm/asciidoctor-all.spec.js

```diff
@@ -21,5 +37,76 @@ describe('Node.js', function () {
       expect(doc.attributes.$fetch('docname')).toBe('test.adoc');
     });
   });
+
+  describe('Converting file', function() {
+    it('should be able to convert a file', function() {
+      var expectFilePath = __dirname + '/test.adoc.html';
```

Comment:
Actually, docname should be extensionless. It seems like the version from lib/asciidoctor/opal_ext/file.rb does not accept the second argument. Fortunately, this method is now implemented in Opal core, so we can either remove this patch or we need to implement the second argument.

See https://github.com/asciidoctor/asciidoctor/blob/v1.5.4/lib/asciidoctor/helpers.rb#L179-L196